REFERENCE.md

```diff
@@ -71,146 +59,151 @@ you know the security tradeoffs of ``FIPS-140-2`` compliance.
 ``FIPS`` mode disables the use of ``MD5`` and may require weaker ciphers or
 key lengths than your security policies allow.
 
+@see http://simp.readthedocs.io/en/master/security_mapping/components/simplib/cryptographic_protection/control.html SIMP - Security Control Mapping Cryptographic Protection
```

Comment:
https://simp.readthedocs.io/en/master/security_mapping/components/simp/cryptographic_protection/control.html

Reply:
stable in lieu of master